integrations/dbt/bigquery/README.md

```diff
@@ -0,0 +1,31 @@
+# dbt-bigquery-openlineage
+
+Plugin for [dbt](https://github.com/fishtown-analytics/dbt) (data build tool) for automatic metadata and lineage collection.
+
+This integration is experimental, and there is high possibility that it will change in the future, 
+as together with dbt developers we're figuring what's the best way to extract metadata from it.
+
+## Requirements
+
+```
+Python 3.6+
+sqlparse>=0.3.1
+```
+
+## Installation
+
+`pip install dbt-openlineage-bigquery`
```

Comment:
I think `pip` cmd should use `openlineage-dbt-bigquery`?

Reply:
Done.